src/errors/Errors.ts

```diff
@@ -167,8 +165,20 @@ export function isBadRequestException(e: AWSError): boolean {
  * Is the exception a retriable exception?
  * @param e The client error caught.
  * @returns True if the exception is a retriable exception. False otherwise.
+ * 
+ * @internal
  */
 export function isRetriableException(e: AWSError): boolean {
```

Comment:
Given that these are no longer exported, we're probably free to change the spelling?

Reply:
Perhaps adding a separate PR for cosmetic items, as Ian suggested below (dates and spelling changes).